### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by lzap**
Do you see any problem @domcleal? Looks sane... This is for 3.0.0

### CODE REVIEW

**Comment by domcleal** [comment]
@lzap have you also extracted them recently?  I can't see it...

**Comment by lzap** [comment]
Yeah, no new strings for the 3.0 :-P

**Comment by lzap** [comment]
May I ask for a quick merge @orrabin ? Looks like all is set for 3.0.0.

**Comment by lzap** [comment]
@domcleal just to confirm, I did:

```
$ rake plugin:gettext[foreman_plugin]
```

No changes in the POT file...

**Comment by domcleal** [comment]
s/foreman_plugin/foreman_discovery/ and I think you may see the timestamp inside the POT update at least.

But yeah, I think you're right, in which case it's OK.

**Comment by lzap** [comment]
Yeah thanks for confirmation, saw those timestamps and things, but I was only interested in the POT updates. So I fetched from Transiflex and it looks we have three languages complete now. Thanks! Ori, good to merge.

File: locale/de/foreman_discovery.po [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 # 
 # Translators:
 # Arnold Bechtoldt <mail@arnoldbechtoldt.com>, 2014
-# Ettore Atalan <atalanttore@googlemail.com>, 2014
+# Ettore Atalan <atalanttore@googlemail.com>, 2014-2015
 # simon11 <simon.stieger.98@live.de>, 2014
 msgid ""
 msgstr ""
-"Project-Id-Version: foreman_discovery 1.4.0.rc2\n"
+"Project-Id-Version: foreman_discovery 2.0.0.rc1\n"
```
Comment (orrabin) [inline locale/de/foreman_discovery.po:11]: is it still 2.0.0 here?

Comment (domcleal) [inline locale/de/foreman_discovery.po:11]: This plugin's version number still says 2.0.0.rc1, so it'll use that.

Comment (orrabin) [inline locale/de/foreman_discovery.po:11]: thanks, merging

**Comment by orrabin** [comment]
thanks @lzap! merged as de513a059bf24f4733746fe7bf3b76ded24d20ce

**Pull request closed without merging**